src/app/js/user/index.js

```diff
@@ -334,6 +335,14 @@ export const getExportPublishedDatasetRequest = (
         url: `/api/export/${type}?${queryString}`,
     });
 
+export const getReorderFieldRequest = (state, fields) =>
+    getRequest(state, {
+        method: 'PUT',
+        url: `/api/field/reorder?${qs.stringify({
```

Comment:
Using query string in this case seems so weird to me..
Any particular reason not to use JSON body?